components/event-publisher-proxy/pkg/legacy/legacy.go

```diff
@@ -240,18 +239,18 @@ func (t *Transformer) convertPublishRequestToCloudEvent(appName string, publishR
 }
 
 // combineEventNameSegments returns an eventName with exactly two segments separated by "." if the given event-type
-// has more than two segments separated by "." (e.g. "Account.Order.Created" becomes "AccountOrder.Created")
+// has more than two segments separated by "." (e.g. "Account.Order.Created" becomes "AccountOrder.Created").
 func combineEventNameSegments(eventName string) string {
 	parts := strings.Split(eventName, ".")
-	if len(parts) > 2 {
+	if len(parts) > 1 {
```

Comment:
what is the reason of changing this ? the comment says also `as more than two segments`

Reply:
i fixed the comment. 
`> 2` is marked by the linter as a `magic number`
the code works exactly the same with `> 1` so that change make sense anyway